docs/design/analytics.md

```diff
@@ -51,7 +51,7 @@ Note: There's a limit of 20 custom dimensions.
 | 5 | `Flag: --style` | `string` |
 | 6 | `--collection` | `string` |
 | 7 | `Flag: --strict` | `boolean` |
-| 8 | `AOT Enabled` | `boolean` |
+| 8 | `Angular CLI Major Version` | `boolean` |
```

Comment:
Should the type be a string (or a number)?

Reply:
Good catch

Reply:
@IgorMinar, would you prefer the value to be a `string` or `number`? As we are uncertain what works bests for filter and querying in GA.

Reply:
I'm actually not sure. It seems that Google analytics converts everything to strings for querying. Let's go with number and we can adjust.